### PR TITLE
Support overrides for the ModelView redirects.

### PR DESCRIPTION
**Pull request by bkyryliuk**
Implementation of the https://github.com/dpgaspar/Flask-AppBuilder/issues/318.
Tested via `nosetests flask_appbuilder/tests/test_base.py`

Reviewers:
- @dpgaspar 

CC:
- @mistercrunch

### CODE REVIEW

File: flask_appbuilder/tests/test_base.py [hunk 1]
```diff
@@ -408,6 +423,41 @@ def test_model_crud(self):
...
         eq_(model, None)
 
+    def test_model_redirects(self):
+        """
+            Test Model redirects after add, delete, edit
+        """
+        client = self.app.test_client()
+        rv = self.login(client, DEFAULT_ADMIN_USER, DEFAULT_ADMIN_PASSWORD)
+
+        #with open('test.txt', 'rb') as fp:
```
Comment (xrmx) [inline flask_appbuilder/tests/test_base.py]: leftover?

Comment (bkyryliuk) [inline flask_appbuilder/tests/test_base.py]: yep

**Comment by bkyryliuk** [comment]
@xrmx, @dpgaspar - https://github.com/dpgaspar/Flask-AppBuilder/blob/master/docs/customizing.rst is it a good location for the docs?

**Comment by xrmx** [comment]
If you add docstrings they should appear in the api reference here: http://flask-appbuilder.readthedocs.io/en/latest/api.html

**Comment by bkyryliuk** [comment]
@xrmx, @dpgaspar - added basic docstrings

File: flask_appbuilder/tests/test_base.py [hunk 2]
```diff
@@ -120,6 +120,20 @@ class Model1View(ModelView):
         class Model1CompactView(CompactCRUDMixin, ModelView):
             datamodel = SQLAInterface(Model1)
 
+        class Model1ViewWithRedirects(ModelView):
+            datamodel = SQLAInterface(Model1)
+            obj_id = 1
+
+            def post_add_redirect(self):
+                return redirect('model1viewwithredirects/show/{}'.format(REDIRECT_OBJ_ID))
```
Comment (xrmx) [inline flask_appbuilder/tests/test_base.py]: {0} so they work in python2.6

File: flask_appbuilder/tests/test_base.py [hunk 3]
```diff
@@ -408,6 +423,39 @@ def test_model_crud(self):
...
+                             data=dict(field_string='test_redirect', field_integer='1',
+                                       field_float='0.12',
+                                       field_date='2014-01-01'), follow_redirects=True)
+
+        eq_(rv.status_code, 200)
+        data = rv.data.decode('utf-8')
+        ok_('Test Redirects' in data)
+
+        model_id = self.db.session.query(Model1).filter_by(field_string='test_redirect').first().id
+        rv = client.post('/model1viewwithredirects/edit/{}'.format(model_id),
```
Comment (xrmx) [inline flask_appbuilder/tests/test_base.py]: {0} there and below

**Comment by bkyryliuk** [comment]
@xrmx - done

**Comment by dpgaspar** [comment]
Thanks! this is merged.

**Pull request closed without merging**